openshift_tools/monitoring/pminfo.py

```diff
@@ -115,6 +118,39 @@ def connect(self):
         """ Establish a PMAPI context to archive, host or local, via args """
         self.context = pmapi.pmContext()
 
+def calculate_percent_cpu(start, end, interval):
+    ''' Calculate the percent of cpu utilization from sample1 and sample2.
+        Requires:
+          start
+          end
+          number of cpus
+          interval
+    '''
+    num_cpu = multiprocessing.cpu_count()
+    jiffy = os.sysconf(os.sysconf_names['SC_CLK_TCK'])
+    return ((end - start) * 100 / (float(jiffy) * interval)) / (num_cpu * 10)
+
+def get_sampled_data(metrics, interval, count=1):
+    '''Recieve a list of metrics and a time interval.
+       sample the metrics, sleep, sample, and then calculate the values.
+    '''
+    if not metrics:
+        return {}
+
+    sampled_results = []
+    for i in range(count):
+        sampled_results.append(get_metrics(metrics))
+        if i == count-1:
+            break
+        time.sleep(interval)
```

Comment:
@twiest,

I think to stay consistent with sar:
       -i interval
              Select data records at seconds as close as possible to the number specified by the interval parameter.

From the man page:
The  accounting  system, based  on  the  values in the count and interval parameters, writes information the specified number of times spaced at the specified intervals in seconds.